logs-lib/README.md

```diff
@@ -0,0 +1,140 @@
+# Logs lib
+
+This logs lib can be used to generate logs dashboard using NEW [grafonnet](https://github.com/grafana/grafonnet).
+
+## Import
+
+```sh
+jb init
+jb install https://github.com/grafana/jsonnet-libs/logs-lib
+jb install https://github.com/grafana/grafonnet/gen/grafonnet-latest@main
```

Comment:
This lib depends upon grafonnet, so this explicit reference isn't really necessary.

Reply:
good one